website/content/docs/discovery/dns.mdx

```diff
@@ -467,9 +467,9 @@ using the [`advertise-wan`](/docs/agent/config/cli-flags#_advertise-wan) and
 [`translate_wan_addrs`](/docs/agent/config/config-files#translate_wan_addrs) configuration
 options.
 
-## Namespaced/Partitioned Services <EnterpriseAlert inline />
+## Namespaced/Partitioned Services and Nodes <EnterpriseAlert inline />
 
-Consul Enterprise supports resolving namespaced and partitioned services via DNS.
+Consul Enterprise supports resolving namespaced and partitioned services and nodes via DNS.
```

Comment:
```suggestion
The DNS server in Consul Enterprise can resolve services assigned to namespaces, partitions, and nodes via DNS.
```
I think this is clearer, but is it still accurate?

Reply:
I reworded it more to make it clear services are namespace+partition and nodes are only partition.